Dockerfile

```diff
@@ -1,17 +1,19 @@
 FROM node:5.7.1-slim
 MAINTAINER Tim Levett <tim.levett@wisc.edu>
-RUN npm install -g grunt-cli superstatic
+RUN npm install -g superstatic
 
 RUN apt-get update && apt-get install -y bzip2 && apt-get install -y git
 
 # Add frame
+COPY tools /build/tools
+copy docs /build/docs
```

Comment:
uppercase `COPY` is more [conventional](https://docs.docker.com/engine/reference/builder/#/format)